### PR TITLE
Adds Craftable Bone Weapons

### PR DESCRIPTION
**Pull request by TehZombehz**
Adds craftable bone dagger, spear, and axe.
Because I don't do balancing, I need opinions on what the stats for these weapons should be and what the crafting recipes should be in order to be properly balanced. Initial stats were minimum-effort suggestions by Kor.
-Bone Dagger = Survival Dagger
-Bone Axe = Fire Axe
-Bone Spear > Glass Spear : has higher damage penetration and more damage in general,
bonewepstogether bonedaggerstand
bonespearstand bonespearstandtw
boneaxestand boneaxestandtw

### CODE REVIEW

File: code/game/objects/items/weapons/twohanded.dm [hunk 1]
```diff
@@ -511,4 +513,61 @@
...
+ * Bone Axe
+ */
+/obj/item/weapon/twohanded/fireaxe/boneaxe  // Blatant imitation of the fireaxe, but made out of bone.
+	icon_state = "bone_axe0"
+	name = "bone axe"
+	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/update_icon()  //Currently only here to fuck with the on-mob icons.
+	icon_state = "bone_axe[wielded]"
+	return
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/twohanded.dm]: trailing return remove it

File: code/game/objects/items/weapons/twohanded.dm [hunk 2]
```diff
@@ -511,4 +513,61 @@
...
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] axes \himself from head to toe! It looks like \he's trying to commit suicide..</span>")
+	return (BRUTELOSS)
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/afterattack(atom/A as mob|obj|turf|area, mob/user, proximity)
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/twohanded.dm]: This proc will inherit from the fireaxe, remove it

File: code/game/objects/items/weapons/twohanded.dm [hunk 3]
```diff
@@ -511,4 +513,61 @@
...
+/obj/item/weapon/twohanded/fireaxe/boneaxe  // Blatant imitation of the fireaxe, but made out of bone.
+	icon_state = "bone_axe0"
+	name = "bone axe"
+	desc = "A large, vicious axe crafted out of several sharpened bone plates and crudely tied together. Made of monsters, by killing monsters, for killing monsters."
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/update_icon()  //Currently only here to fuck with the on-mob icons.
+	icon_state = "bone_axe[wielded]"
+	return
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/suicide_act(mob/user)
```
Comment (ChangelingRain) [inline code/game/objects/items/weapons/twohanded.dm]: Same for this proc, it'll inherit

File: code/game/objects/items/weapons/twohanded.dm [hunk 4]
```diff
@@ -511,4 +513,60 @@
...
+	user.visible_message("<span class='suicide'>[user] axes \himself from head to toe! It looks like \he's trying to commit suicide..</span>")
+	return (BRUTELOSS)
+
+/obj/item/weapon/twohanded/fireaxe/boneaxe/afterattack(atom/A as mob|obj|turf|area, mob/user, proximity)
```
Comment (Cheridan) [inline code/game/objects/items/weapons/twohanded.dm]: uh if this is the same as normal axes you don't need to add this, it's inherited

Comment (TehZombehz) [inline code/game/objects/items/weapons/twohanded.dm]: I made this a little quickly, tbh.

File: code/game/objects/items/weapons/twohanded.dm (hunk 3 above)
Comment (Supermichael777) [inline code/game/objects/items/weapons/twohanded.dm]: uhhh not unless you make it inherit the whole proc your overriding the return. 

if your not changing the text just let it inheret its suicide act from its parent.

File: code/game/objects/items/weapons/twohanded.dm [hunk 5]
```diff
@@ -511,4 +513,44 @@
...
+/obj/item/weapon/twohanded/fireaxe/boneaxe/update_icon()
+	icon_state = "bone_axe[wielded]"
+
+/*
+ * Bone Spear
+ */
+/obj/item/weapon/twohanded/bonespear	//Blatant imitation of spear, but made out of bone. Not valid for explosive modification.
+	icon_state = "bone_spear0"
+	name = "bone spear"
+	desc = "A large sharpened bone on the end of a long stick. A standard of ancient tribal warfare."
```
Comment (Supermichael777) [inline code/game/objects/items/weapons/twohanded.dm]: "A haphazardly-constructed yet still deadly weapon. The pinnacle of modern technology."

This is a joke about the normal spears description btw.

**Comment by PKPenguin321** [comment]
what is stopping me from making a gorillion of these every round with the infinite bones/sinew i get from mobs spawning out of tendrils?

**Comment by ToadStar** [comment]
Actually being able to continuously kill the monsters coming out from them.

**Comment by PKPenguin321** [comment]
so nothing then?

**Comment by Supermichael777** [comment]
your ability to effectivly carry that shit. Hauling around 2 spears and an ax will only get you so far

**Comment by PKPenguin321** [comment]
good thing i just happen to be buttbuddies with science and get bags of holding for doing my job

not to mention that i can also give my axes to my friends in security/the revolution and now _everybody_ can have one of the highest brute weapons available to the crew!

**Comment by optimumtact** [comment]
not sure about the fireaxe clone but everything else seems cool

**Comment by Archie700** [comment]
The infinite weapons would depend on a) being able to stand killing a huge amount of monsters, b) actually surviving them, c) hoping some other miner does not kill the tendril for the chest, d) hoping that science has enough research, and e) miners actually bringing back materials.

File: code/game/objects/items/weapons/twohanded.dm [hunk 6]
```diff
@@ -511,4 +513,44 @@
...
+	force = 10
```
Comment (Fox-McCloud) [inline code/game/objects/items/weapons/twohanded.dm]: should match `force_unwielded`

**Comment by Firecage** [comment]
The axe blade is to unrealistically large.

**Comment by yackemflam** [comment]
@Firecage The axe blade seems fine to me

**Comment by AdamElTablawy** [comment]
oh boy more improvised weapons, just what we needed

i really am not a fan of spammable fireaxes, everything else is okay i guess

**Comment by Archie700** [comment]
On balance, I think the bone axe should be weaker with less sharpness, because after some thought, it really isn't good to make multiples of a fire axe, considering how damaging it is.

**Comment by xxalpha** [comment]
These shouldn't be craftable.

File: code/game/objects/items/weapons/twohanded.dm (hunk 6 above)
Comment (Supermichael777) [inline code/game/objects/items/weapons/twohanded.dm]: why doesn't this just use initial?

**Comment by Iamgoofball** [comment]
>no bone club

pls

**Comment by Iamgoofball** [comment]
@xxalpha fuck off

**Comment by KingDairy** [comment]
massive ash drake sword that does burn/brute and sets people on fire when

**Comment by RandomMarine** [comment]
>not a slow, charged swing that ex_acts everything in the tile clicked on.

**Comment by TehZombehz** [comment]
Closing for Kor-oriented maintenance.

**Pull request closed without merging**